README.adoc

```diff
@@ -131,6 +131,24 @@ This can be achived with the {uri-kube-apiserver}[Kubernetes API server CLI], e.
 $ kube-apiserver --cors-allowed-origins .*
 ```
 
+== Authentication
+
+We try to support the various authentication strategies supported by https://kubernetes.io/docs/reference/kubectl/overview/[kubectl] in order to provide seamless integration with your local setup. Here are the different authentication strategies we support depending on how you're using Kubebox:
```

Comment:
kubectl -> `kubectl`

Reply:
Just format kubectl as an inline code block

Reply:
Not sure how to do that in a hyperlink, tried

https://kubernetes.io/docs/reference/kubectl/overview/[[source shell]`kubectl`]

Reply:
https://kubernetes.io/docs/reference/kubectl/overview/[`kubectl`] should work

Reply:
Done, thanks!